onnxruntime/test/providers/cpu/model_tests.cc

```diff
@@ -664,6 +664,8 @@ ::std::vector<::std::basic_string<ORTCHAR_T>> GetParameterStrings() {
       ORT_TSTR("split_zero_size_splits"),
       ORT_TSTR("convtranspose_3d")};
   static const ORTCHAR_T* openvino_disabled_tests[] = {ORT_TSTR("tf_mobilenet_v1_1.0_224"),
+                                                       ORT_TSTR("yolov3"),
```

Comment:
why is yolov3 disabled now?

Reply:
Hey George, 
The yolov3 model that we have in the big zip file, runs fine with onnx_test_runner, but with the new ModelTests which run with onnxruntime_test_all, it's failing for some reason. I was not able to debug this because i couldn't get single model to run with onnxruntime_test_all.  So i disabled it as we can run it with onnx_test_runner accurately. Please let me know if there is a way to run this single model and debug this but yolov3 model is working with test_runner and perf_test

Reply:
onnxruntime_test_all is just using gtest , so I believe you can just add --gtest_filter option to filter to a single test for debugging.

Reply:
I tried that but doesn't work. It needs a folder for build and we have to copy the models directory into that and it just picks each of them and runs them. The gtest name is something like ModelTests/ModelTest.Run/73 and i'm not able to use that as a name for the filter.

Reply:
> I tried that but doesn't work. It needs a folder for build and we have to copy the models directory into that and it just picks each of them and runs them. The gtest name is something like ModelTests/ModelTest.Run/73 and i'm not able to use that as a name for the filter.

@snnn any suggestions on how to debug single model test due to the migration to onnxruntime_test_all ?

Reply:
Model name should be still in the log.

Reply:
When it fails again, please give me a link, I'll show you how to find the model file's path.

Reply:
cool. Let me try to get the error again.

Reply:
Hey @snnn,
So, after the test fails, i get this. 
[  FAILED  ] ModelTests/ModelTest.Run/147, where GetParam() = "openvino_../models/opset10/yolov3/yolov3.onnx"

But when i try to run  ./onnxruntime_test_all --gtest_filter="ModelTests/ModelTest.Run/147"  individually, it passes for me. 
==========] Running 1 test from 1 test suite.
[----------] Global test environment set-up.
[----------] 1 test from ModelTests/ModelTest
[ RUN      ] ModelTests/ModelTest.Run/147
[       OK ] ModelTests/ModelTest.Run/147 (1917 ms)
[----------] 1 test from ModelTests/ModelTest (1918 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test suite ran. (1918 ms total)
[  PASSED  ] 1 test.

Reply:
You can try to get the model and run it with onnx_test_runner, usually it should be same. 
Otherwise, it must because of some shared resource that reused between session. And in that case, unfortunately, it would be harder to figure it out.

Reply:
Hey @jywu-msft, as this model is working individually and also with onnx_test_runner, can we mark this as disabled for model_tests

Reply:
yeah, we can disable it for the sake of this checkin.
But it still feels like there is a bug lurking that needs to be debugged further. 
Would you be able to add more instrumentation/debug output to show what the failure is and where it's happening?